packages/footer/components/Navigation.tsx

```diff
@@ -1,18 +1,102 @@
 import * as React from 'react';
 import {
    Box,
-   BoxProps,
    forwardRef,
    List,
    ListIcon,
    ListItem,
-   ListItemProps,
    ListProps,
    SimpleGrid,
+   Text,
+} from '@chakra-ui/react';
+import type {
+   BoxProps,
+   ListItemProps,
    SimpleGridProps,
 } from '@chakra-ui/react';
+import { NewsletterComponent } from './Newsletter';
+import type { NewsletterFormProps } from './Newsletter';
+import type { Menu as MenuType, MenuItem } from '@ifixit/ui';
 import { useTrackedOnClick } from '../hooks/useTrackedOnClick';
 
+type NavSectionProps = {
+   menu1: MenuType | null;
+   menu2: MenuType | null;
+   menu3: MenuType | null;
+   newsletterForm: NewsletterFormProps;
+};
+
+export const NavigationSection = ({
+   menu1,
+   menu2,
+   menu3,
+   newsletterForm,
+}: NavSectionProps) => {
+   return (
+      <FooterNavigationSection p={5}>
+         <FooterNavigationList pl={-5}>
+            <NavigationColumn menu={menu1} />
+         </FooterNavigationList>
+         <FooterNavigationList pl={-5}>
+            <NavigationColumn menu={menu2} />
+         </FooterNavigationList>
+         <FooterNavigationList pl={-5}>
+            <NavigationColumn menu={menu3} />
+         </FooterNavigationList>
+         <NewsletterComponent newsletterForm={newsletterForm} />
+      </FooterNavigationSection>
+   );
+};
+
+const NavigationColumn = ({ menu }: { menu: MenuType | null }) => {
+   console.log('Menu: ', menu);
+
+   if (!menu) {
+      return null;
+   }
+   return (
+      <>
+         <Text fontSize="14px" fontWeight="bold" color="white" my={2}>
```

Comment:
```suggestion
         <Text fontSize="16px" fontWeight="bold" color="white" my={2}>
```

This would match the footer on the homepage

**Preview**
![preview](https://user-images.githubusercontent.com/95656772/221929710-423a75d6-9cee-4513-baf0-0050a3992140.jpg)

**Homepage**
![prod](https://user-images.githubusercontent.com/95656772/221929721-02aace67-baba-4127-8765-f43134a15685.jpg)

Reply:
The homepage footer will soon be replaced with an import of this footer, and the figma is meant to be the canonical for both of these. I'm fine with this change, and I personally think the larger text looks nicer for the header.

Should we stick with the figma design or make this change?

Reply:
Actually, I think it might be more difficult to use this component now that we are going to use strapi to populate it, so I'll match the homepage size to keep it consistent.